code/modules/clothing/spacesuits/flightsuit.dm

```diff
@@ -492,6 +520,38 @@
 		losecontrol(move = FALSE)
 	crashing = FALSE
 
+/obj/item/device/flightpack/proc/door_hit(obj/structure/mineral_door/door)
```

Comment:
ehhhhhhhhhhh I'm not sold on that Anturk, I can't see many things needing to interact with doors like this